src/Service/AuthScreenService.php

```diff
@@ -0,0 +1,136 @@
+<?php
+
+namespace App\Service;
+
+use App\Entity\Screen;
+use App\Entity\ScreenUser;
+use Doctrine\ORM\EntityManagerInterface;
+use Lexik\Bundle\JWTAuthenticationBundle\Services\JWTTokenManagerInterface;
+use Symfony\Component\Uid\Ulid;
+use Symfony\Contracts\Cache\CacheInterface;
+
+class AuthScreenService
+{
+    public const BIND_KEY_PREFIX = 'BindKey-';
+
+    public function __construct(
+        private CacheInterface $cache,
```

Comment:
Fix parameter naming for autowiring (https://symfony.com/doc/current/cache.html#creating-custom-namespaced-pools)